src/main/java/edu/brown/cs/student/main/StarDistance.java

```diff
@@ -0,0 +1,43 @@
+package edu.brown.cs.student.main;
+
+public class StarDistance {
+  /**
+   * Default constructor
+   */
+  public StarDistance() {
+
+  }
+
+  /**
+   * Method that calculates the Euclidean distance between two coordinates in 3D space
+   *
+   * @param coordinate1 - Double[3]
+   * @param coordinate2 - Double[3]
+   * @return distance in double
+   */
+  public double euclideanDistance(Double[] coordinate1, Double[] coordinate2) {
+    return Math.sqrt(Math.pow(coordinate1[0] - coordinate2[0], 2)
+        + Math.pow(coordinate1[1] - coordinate2[1], 2)
+        + Math.pow(coordinate1[2] - coordinate2[2], 2));
+  }
+
+  /**
```

Comment:
The use of java docs is great for readability and ensuring future changes can be made easily